petModule/Pet.java

```diff
@@ -0,0 +1,308 @@
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.concurrent.TimeUnit;
+
+public class Pet {
+    /* Instance Variables */ 
+    private String name;
+    final private String ownerID; 
+    private int trustLevel; //-10 to 10
+    private int moodLevel; //-10 to 10
+    private String color;
+    private int[] mStates; //Maintenance State field
+    private int daysAtWorstTrust; //Need way to keep track of days at -10
+    private Date lastDAWT; //Need way to keep track of days at -10
+    final private Date birthDate; // possible birthday tracker?
+
+    /* Constructor */
+    public Pet(String userID) {
+        /*TODO: get user's name */
+        String userName = "Joe Mama";
+        this.name = userName + "\'s" + " pet";
+        
+        this.ownerID = userID;
+        
+        this.trustLevel = 0;
+
+        this.moodLevel = 0;
+
+        this.color = "default";
+
+        this.mStates = new int[] {0, 0};
```

Comment:
these should be individual booleans instead of being wrapped in an int array, unless they store some amount of time since the last time it was fed/bathed

Reply:
You're totally right, they don't need to be arrays. Don't know why I wrote them as int arrays. They're separate booleans now.